<?xml version="1.0"?>
<launch>
    <arg name="namespace" default="tello" />
    <arg name="use_wifi" default="False" />
    <arg name="bluetooth_addr" default="e0:14:ef:39:3d:d1" />

    <group ns="$(arg namespace)">
        <node pkg="tello_driver" name="tello" type="tello_driver_node.py" output="screen">
        </node>
    </group>
</launch>